EventFilter/DTRawToDigi/python/dturosunpacker_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+dturosunpacker = cms.EDProducer("DTuROSRawToDigi",
+                                  inputLabel = cms.InputTag("source"),
+                                  feds     = cms.untracked.vint32( 1369, 1370, 1371,),
```

Comment:
Those numbers are also hardcoded in ```DTuROSRawToDigi::theDDU```: no need to make them configurable, as if you choose different ones you'll end up in some misbehaviour.
I think you should simply modify the producer removing the possibility to configure them